loops.js

```diff
@@ -10,8 +10,10 @@
 // Tip: je kunt de .repeat() methode gebruiken om een karakter een n aantal keer te herhalen...
 // Dit heb je nog niet geleerd, maar bekijk hiervoor dit MDN-artikel eens: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/repeat
 // ==========================================
-
-
+for(let i=1; i <= 5; i++){
+    let a = "*".repeat(i);
```

Comment:
De implementatie om een driehoek van sterretjes te printen is eenvoudig en effectief door gebruik te maken van "*".repeat(i). Dit zorgt voor een duidelijke en begrijpelijke opbouw.